barman/backup.py

```diff
@@ -589,6 +595,30 @@ def delete_backup(self, backup, skip_wal_cleanup_if_standalone=True):
 
         return True
 
+    def _set_backup_sizes(self, backup_info, fsync=False):
+        """
+        Set the actual size on disk of a backup.
+        Optionally fsync all files in the backup.
+
+        :param LocalBackupInfo backup_info: the backup to update
+        :param bool fsync: whether to fsync files to disk
+        """
+        backup_size = 0
+        deduplicated_size = 0
+        backup_dest = backup_info.get_basebackup_directory()
+        for dir_path, _, file_names in os.walk(backup_dest):
+            if fsync:
+                fsync_dir(dir_path)
+            for filename in file_names:
+                file_path = os.path.join(dir_path, filename)
+                file_stat = fsync_file(file_path) if fsync else os.stat(file_path)
+                backup_size += file_stat.st_size
+                if file_stat.st_nlink == 1:
```

Comment:
```suggestion
                if file_stat.st_nlink == 1:
```

Here I'd say to add the comment back regarding what it means, just so it's handy.